packages/core/src/components/Onboarding/index.tsx

```diff
@@ -36,6 +37,14 @@ export function Onboarding({
 
   const [page, setPage] = useState<number>(platform === 'web' ? 1 : 0)
 
+  let names = namer(primaryColor)
+
+  useEffect(() => {
+     names = namer(primaryColor)
+    setPrimaryName(names.html[0].name)
```

Comment:
I know that the library guarantees that `names.html[0].name` would always exist, but maybe we could use optional chaining to ensure that this value exists and the page doesn't error out if some of these values are undefined.
Again, I don't think it's necessary but it's just a nit from my Typescript love. What are your thoughts @alexdanilowicz ?

Reply:
yeah, I think we should do somechaining and just set it to a blank string worse case. or wrap in a try catch